client/src/helpers/index.js

```diff
@@ -118,6 +118,27 @@ export const formatDatewTimeZoneWeekdayShort = (ts, timeZone) => {
   });
 };
 
+export const formatShortWeekdayToLong = (shortWeekday) => {
```

Comment:
Nit: this function is straightforward, we can simplify the function using an object to map the short names to their counterparts.

```
export const formatShortWeekdayToLong = (shortWeekday) => {
  const weekdays = {
    Mon: "Monday",
    Tue: "Tuesday",
    Wed: "Wednesday",
    Thu: "Thursday",
    Fri: "Friday",
    Sat: "Saturday",
    Sun: "Sunday",
  };

  return weekdays[shortWeekday] || null;
};
```